hack/install.sh

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
```

Comment:
Add a usage of this script please.

Reply:
usage will add in our docs：https://github.com/karmada-io/website/pull/205